pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -946,8 +946,8 @@ func aceToRules(aclArgs types.AppNetworkACLArgs, ace types.ACE) (types.IPTablesR
 	aclRule3.IsUserConfigured = true
 	aclRule3.RuleID = ace.RuleID
 	if aclArgs.NIType == types.NetworkInstanceTypeSwitch {
-		if len(aclArgs.UpLinks) > 1 {
-			errStr := fmt.Sprintf("aceToRules: Switch network instance with more than ONE " +
+		if len(aclArgs.UpLinks) != 1 {
+			errStr := fmt.Sprintf("aceToRules: Switch network instance not with only ONE " +
```

Comment:
This reads weird. Should it be replaced with "Switch network instance is only supported with exactly one port attached for now".

Reply:
Actually the change is plain wrong. We support a switch NI with no external ports; only to connect app instances.